### PR TITLE
Js update readme

### PR DESCRIPTION
**Pull request by jordy1611**
Adds frontend images and user details to readme

### CODE REVIEW

**Review by Leasw144 — APPROVED** [review]
Thanks for getting the images up

File: README.md [hunk 1]
```diff
@@ -24,6 +24,22 @@ BE:
...
 
+## Survey Display
+
+![Top of Survey](./readme_assets/surveyTop.png)
+
+![Middle of Survey](./readme_assets/surveyMid.png)
+
+![Bottom of Survey](./readme_assets/surveyBot.png)
+
+![Finished Survey](./readme_assets/surveyDone.png)
```
[the remark this answers is not in the document]
Comment (janegreene) [reply README.md:35]: Covid Couture? Of course! Thanks for the image updates.

**Review by janegreene — APPROVED** [review]
Nice updates.

**Pull request merged**